### ISSUE TITLE
Blood Tap activating 2 Death runes

### ISSUE THREAD

**Issue by Shragenator** [SunwellTracker/issues#2946]
Decription: 
Using the blood tap ability in a certain situation causes the ability to instantly activate as death runes the 2 runes that are on cooldown in the blood rune slot.

How it works: 
When specced into "Blood of the North" one can convert blood runes into death runes.  In the situation where 1 death rune (in a blood rune slot) AND 1 blood rune are on cooldown simultaneously, using "Blood Tap" will instantly activate both these runes as death runes.

How it should work: 
If a blood rune and a death rune (in a blood rune slot) are on cooldown, activating blood tap should instantly activate only the blood rune as a death rune, while leaving the other death rune still on cooldown. 

Source (you should point out proofs of your report, please give us some source): 
In game, does not match the tooltip.

**Comment by sirjokero** [comment]
i just tested and he is right

**Comment by Markbyte123** [comment]
This is working as intended, blood tap runs 2 effects, one is rune conversion, the other is rune refresh, they are separate from each other, but follow the 125634 rule, there is no bug here

**Comment by Markbyte123** [comment]
Blood tap should do a rune conversion and rune refresh at the same time in order of 125634, looks like there are some hard coded examples, def some bugs with it from iteration to iteration, try coding like mentioned above

**Comment by avarishd** [comment]
Yep, 1 year old bug, still no new, nothing. How to replicate. spec  [Blood of the North] 3/3. Then do Icy Touch => Plague Strike => obliterate  => Pestilence => Blood Tap and then you can see you get 2 death runes (fully charged), instead of the blood rune that was meant to be turned into a death rune (the one on cooldown), you get 2 death runes.

**Comment by avarishd** [comment]
@Wojtyas Yep, 1 year old bug, still no new, nothing. How to replicate. spec  [Blood of the North] 3/3. Then do 
Icy Touch => Plague Strike => obliterate  => Pestilence => Blood Tap 
And then you can see you get 2 death runes (fully charged), instead of the one BR that was supposed to be turned into a death rune, you get 2.

**Comment by avarishd** [comment]
Actually, no, this is the same bug I reported years ago https://github.com/SunwellTracker/issues/issues/2158 @Azathothh @Dootz

**Comment by avarishd** [comment]
And this is me simply using blood strike => rune tap

https://user-images.githubusercontent.com/46330494/111002686-867f0c80-838e-11eb-8ef8-c70fd4d5403c.mp4

**Comment by avarishd** [comment]
@Wojtyas Sorry for tagging but this was reported over a year ago...

**Comment by Shragenator** [comment]
Keep in mind @avarishd that if your second blood rune was on cd there it would also return it as a death rune (so if you used D&D or blood boil before using blood tap)

**Comment by avarishd** [comment]
> Keep in mind @avarishd that if your second blood rune was on cd there it would also return it as a death rune (so if you used D&D or blood boil before using blood tap)

Yeah I know but they are still not fixing this thing...its over a fucking year old
